tests/unit/test_sqlite_storage.py

```
@@ -491,6 +491,44 @@ async def test_keyword_search_ignores_archived_memories(self, adapter):
         results = await adapter.keyword_search("archived keyword search test", top_k=10)
         assert all(r.memory.archived_at is None for r in results)
 
+    async def test_keyword_search_matches_non_adjacent_words(self, adapter):
+        """マルチワードクエリが非隣接でもマッチすること（暗黙 AND）。"""
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)